src/compiler/gcc.rs

```diff
@@ -169,7 +176,7 @@ counted_array!(pub static ARGS: [ArgInfo<ArgData>; _] = [
     flag!("-fplugin=libcc1plugin", TooHardFlag),
     flag!("-fprofile-arcs", ProfileGenerate),
     flag!("-fprofile-generate", ProfileGenerate),
-    take_arg!("-fprofile-use", OsString, Concatenated, TooHard),
+    take_arg!("-fprofile-use", PathBuf, Concatenated('='), ProfileUse),
```

Comment:
`-fprofile-use` can also be used without a value associated with it. It looks like that's true of `-fprofile-instr-use` too, and you don't handle those cases. There's also `-fno-profile-instr-use`, which you might as well add support for while you're here.

Reply:
It looks like `-fprofile-use` without argument will try to read from `./default.profdata` -- and that the implementation here already handles this correctly. I'll add a unit test for that.